main-app/MoveMaster-app/Assets/Scripts/Scoreboard/ScoreboardManager.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class ScoreboardManager : MonoBehaviour
+{
+    public static ScoreboardManager Instance;
```

Comment:
Dałabym w sumie nazwę mówiącą czego to jest instancja

Reply:
Nie zgadzam się.

Nazwa Instance wynika z konceptu design patternu singleton

Reply:
Nie do poprawy